parametric/build.sbt

```
@@ -0,0 +1,45 @@
+import BenchExecDsl._
+import ProjectUtils._
+
+enablePlugins(BenchExec)
+
+benchmarks ++= Seq(
+  suiteGen("003parametric-apalache", parametricSpecs, cmdGenGen())
+)
+
+lazy val parametricSpecs = Seq(
+  Spec("parametric-specs", "SetAdd.tla", inv = "Inv"),
+  Spec("parametric-specs", "SetAddDel.tla", inv = "Inv"),
+  Spec("parametric-specs", "SetSndRcv.tla", inv = "Inv"),
+  Spec("parametric-specs", "SetSndRcv_NoFullDrop.tla", inv = "Inv"),
+)
+
+def cmdGenGen() = {
```

Is this meant to have `Gen` twice in the name, or should it be

```suggestion
def cmdGen() = {
```

---

This generates generators, I will add a comment about it :-)